packages/platforms/mellanox/x86-64/x86-64-mlnx-msn2700/onlp/builds/src/module/src/debug.c

```diff
@@ -0,0 +1,45 @@
+#include "x86_64_mlnx_msn2700_int.h"
```

Comment:
This debug interface is deprecated (in favor of the onlp_sysi_debug() entry point). 
I know its just copy-pasted but you can remove this entire file.